### ISSUE TITLE
AffordanceNet architecture implementation

### ISSUE THREAD

**Issue by apicis** [beapc18/AffordanceNet#2]
Hi, I have few questions about the AffordanceNet implementation:

* Could you please list the differences or design choices with respect to the original Caffe implementation?
* Concerning the available checkpoint, was the initial model or some of its component (e.g. VGG) pretrained on COCO or some other dataset?
* In the README the results of AffordanceNet are the ones in the paper or did you run again the model installing Caffe?
* What are the differences between the performance measure of this repository and the one used in the original paper? 

Thank you very much

**Comment by beapc18** [comment]
Hi @apicis, 
- There are no differences in the architectures, they are as the original networks
- Only the VGG component is pretrained on imagenet, the keras VGG model is directly used with the option by default `weights="imagenet"`
- The results under the column `AffordanceNet (Caffe)` are from the paper, and under the column `AffordanceNet*` you can find the results of running their model (installing Caffe) but using the evaluation python code that I used for the tf architectures.
- The performance measure is the same, weighted F-measure. The variance in the final metrics come from some difference in the code used in the experiments. I couldn't find their code to apply the exact same evaluation, that's why I included the column in the middle, `AffordanceNet*`, to compare the performance using my evaluation code with the results in the papers.

Thank you for your questions, hope this helps :)

**Comment by apicis** [comment]
Thank you very much for your reply and for your Tensorflow implementation @beapc18. 

* Do you happen to have the files with the test outputs (classes, boxes and masks) of the original AffordanceNet (Caffe)?
* The script ``affordancenet_predictor.py`` performs 10 predictions and outputs the Fwbeta and mAP scores for these first 10 predictions. Do you have the file containing the annotations also for the other parts of the training set and/or the test set?  

Thanks

**Comment by beapc18** [comment]
I have looked for the files but I don't have them anymore, sorry
